src/coreclr/nativeaot/System.Private.CoreLib/src/Internal/DeveloperExperience/DeveloperExperience.cs

```diff
@@ -134,14 +134,11 @@ private static string GetFileNameWithoutExtension(string path)
 
         private static string GetFileName(string path)
```

Comment:
I would be better to delete GetFileName and GetFileNameWithoutExtension, and replace it with a call to `Path.GetFileNameWithoutExtension`

Reply:
I was going to ask about that, but then figured there was a reason they were being duplicated here.  I guess not?